packages/tools/webpack-fluid-loader/src/routes.ts

```diff
@@ -238,7 +238,10 @@ export const after = (app: express.Application, server: WebpackDevServer, baseDi
         }
 
         const documentId = req.params.id;
-        if (documentId !== "new" && documentId !== "manualAttach") {
+        // For testing alfred, we use the path: http://localhost:8080/testalfred. This will use the local storage
```

Comment:
nit: Can we rename this to "testorderer"?

Reply:
done. I named it to that first and but then switched to testalfred. :)